creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -0,0 +1,54 @@
+const promClient = require('prom-client')
+
+const METRIC_PREFIX = 'cn_'
```

Comment:
i can do this here, but it seems redundant with target-specific labels defined here https://github.com/AudiusProject/audius-protocol/blob/master/monitoring/prometheus/generate-prom.js#L25?

seems like namespace prefix should be used to differentiate different namespaces within a single target, e.g. `server`, `db`, etc?

Reply:
The link you sent was for a label and not the metric name. That label is also solely [`stage` or `prod`](https://github.com/AudiusProject/audius-protocol/blob/master/monitoring/prometheus/generate-prom.js#L8) at the moment.

For sure, we could remove the `audius_` namespace according to the [docs](https://prometheus.io/docs/practices/naming/), but it does make it harder for the org to find existing metrics using the [Grafana Metric Explorer](http://grafana.audius.co/explore). This will get harder to navigate once we add different exporters for different metrics like AWS, GCP, Pingdom, Senty, PD, etc, all of which bring their own unique namespace.

I strongly vote for having an "audius" prefix, but wouldn't be opposed to `aud_` or even `a_` if we're concerned about character count.

Reply:
agreed on prefixing `audius_` bc of the various exporters with various namespaces from a single target
but instead of relying on `audius_dn` or `audius_cn` etc, we should instead use the prometheus defined labels
thanks for linking to the grafana metric explorer - that actually reinforces in my mind the pattern of relying on the prometheus labels to identify service type

sounds like we are in agreement to remove the _cn_ and _dn_ prefix, i'll do that now

Reply:
Heh, I think we were talking about different things. :)

My vote is having an "audius" prefix (`aud_`, `a_`, or `audius_`) before the `cn_` or `dp_` prefix.

The reason for this is to ensure metrics from the "audius_cn" namespace do not conflict with metrics from the "audius_dp" namespace without having to look at /prometheus_metrics and/or both codebases to confirm the absence of a clash.